.circleci/config.yml

```diff
@@ -489,6 +489,48 @@ jobs:
           file: build/logs/clover.xml
       - save-npm-cache
 
+  behat-absolute-url-coverage:
```

Comment:
Why do we need a separate job for this?

Reply:
If we do this for every feature we'll have a combinatorial explosion.

Reply:
I added that because it is a configuration that changes the output of all API responses (IRIs). From `/api/resource/1` to `http://example.com/api/resource/1`. If you have any other ideas on how I could set a configuration for a particular test, I'd be happy to implement.

Reply:
We only need a test for it, not a separate job. :smile:

Reply:
I see you've duplicated a part of our test suite. I think that's overkill. :see_no_evil: